5.0/fr/0x10-V1-Encoding-and-Sanitization.md

```diff
@@ -0,0 +1,103 @@
+# V1 Encodage et nettoyage
+
+## Objectif du contrôle
+
+Ce chapitre aborde les failles de sécurité les plus courantes des applications web liées au traitement non sécurisé de données non fiables. Ces failles peuvent entraîner diverses vulnérabilités techniques, où les données non fiables sont interprétées selon les règles syntaxiques de l'interpréteur concerné.
```

Comment:
"untrusted data" / "non fiables" → I don't like this one but maybe it's the authorized wording for this concept in french?